docs/profiling.md

```diff
@@ -0,0 +1,92 @@
+# vHive profiling tool
+
+## Methodology
+
+The tool is for benchmarking vHive instances by recording hardware counters, requests per second
+ (RPS) and execution time for functions running inside instances.
+
+The tool includes following components: 
+- A loader function loads requests to vHive round by round (increment 5% of input RPS each round 
```

Comment:
the step (call it a step not increment) is configurable, you should say it: "a step, e.g., 5% of the ...". "input RPS" is unclear

Reply:
Done